### PR TITLE
COMMERCE-6081 - COMMERCE-6078

### PR DESCRIPTION
**Pull request by gianmarcobrunialti**
This includes minor fixes for:
* Account Selector FE component
* AddToWishList FE component

### CODE REVIEW

**Comment by gianmarcobrunialti** [comment]
ci:test:sf

**Comment by gianmarcobrunialti** [comment]
ci:test:relevant

**Comment by ivicac** [comment]
ci:test:relevant

**Comment by gianmarcobrunialti** [comment]
ci:test:sf

**Comment by gianmarcobrunialti** [comment]
ci:test:relevant

File: modules/apps/commerce/commerce-frontend-js/src/main/resources/META-INF/resources/components/account_selector/views/OrdersListView.js [hunk 1]
```diff
@@ -60,9 +61,7 @@ function OrdersListView({
 						if (!items || !items.length) {
 							return (
 								<EmptyListView
-									caption={Liferay.Language.get(
-										'no-orders-were-found'
-									)}
+									caption="no-orders-were-found"
```
Comment (ivicac) [inline modules/apps/commerce/commerce-frontend-js/src/main/resources/META-INF/resources/components/account_selector/views/OrdersListView.js:64]: @gianmarcobrunialti does it mean, here we will not have anymore localization

Comment (gianmarcobrunialti) [reply modules/apps/commerce/commerce-frontend-js/src/main/resources/META-INF/resources/components/account_selector/views/OrdersListView.js:64]: @ivicac For the `EmptyListView` component, i18n is centralized.
image

**Review by ivicac — APPROVED** [review]

**Comment by ivicac** [comment]
ci:forward

**Pull request closed without merging**